docs/virtual_machine_template_guide.adoc

```diff
@@ -0,0 +1,189 @@
+= VirtualMachine Templates
+
+VirtualMachine templates are openshift templates (https://docs.openshift.org/latest/dev_guide/templates.html) +
+that describe VirtualMachine specifications which produce the following template objects: +
+**One** OfflineVirtualMachine and PersistentVolumeClaim. +
```

Comment:
If we stated how many offline vms is produced I think we should add cardinality for pvc(s) as well.

Reply:
I think so too,
@fabiand  do we want to restrict the PVC to one currently ?   or mention a 1-N cardinality here ?

Reply:
since we can't specify templates (e.g. size) for a single PVC, we can allow adding an arbitrary number.